src/ports/postgres/modules/utilities/path.py_in

```diff
@@ -108,8 +107,11 @@ def path(schema_madlib, source_table, output_table, partition_expr,
                             *,
                             nextval('{seq_gen}') AS {id_col_name},
```

Comment:
why not simply `row_number() over ()`? (no need to create and drop the sequence)